creators_custom/RoutesCreatorFenix.py

```diff
@@ -0,0 +1,501 @@
+
+        save_routes_to_file(routes)
+
+    def get_stops_of_route(self, route_id):
+        api = overpy.Overpass()
+
+        result = api.query("""
+        <id-query ref="%s" type="relation"/>
+        <print/>
+        """ % str(route_id))
+
+        if len(result.get_relations()) == 0:
+            raise RuntimeError(
+                "Route not found: https://www.openstreetmap.org/relation/" + str(route_id))
+
+        stops = []
+
+        for member in result.get_relation(route_id, resolve_missing=True).members:
+            if member.role.startswith('platform'):
+                stop = self.from_relation_member(member)
+                stops.append(stop)
+            elif CHECK_FOR_NON_PLATFORMS:
+                # It is possible bus_stops are not properly set to role=platform*
+
+                if member.role == "forward" or member.role == "backward":
+                    continue
+
+                print "Looking for bus stops without proper role in relation..."
+
+                complete_member = member.resolve(resolve_missing=True)
+                tags = complete_member.tags
+
+                if isinstance(member, overpy.RelationNode):
+                    if 'highway' in tags and tags['highway'] == 'bus_stop':
+                        stop = self.from_relation_member(member)
+                        stops.append(stop)
+                        sys.stderr.write("Node has no platform* role in relation " +
+                                         str(route_id) + ": https://www.openstreetmap.org/node/" +
+                                         str(member.ref) + "\n")
+                elif isinstance(member, overpy.RelationWay):
+                    if 'amenity' in tags and tags['amenity'] == 'bus_station':
+                        stop = self.from_relation_member(member)
+                        stops.append(stop)
+                        sys.stderr.write("Way has no platform* role in relation " + str(route_id) +
+                                         ": https://www.openstreetmap.org/way/" + str(member.ref) +
+                                         "\n")
+
+        return stops
+
+    def save_routes_to_file(self, outes):
+        routes = creators_custom.ROUTES
+        with open('data/routes.pkl', 'wb') as f:
+            pickle.dump(routes, f, pickle.HIGHEST_PROTOCOL)
+
+    def read_routes_from_file(self):
+        if os.path.isfile('data/route_masters.pkl'):
+            with open('data/routes.pkl', 'rb') as f:
+                routes = pickle.load(f)
+            return routes
+        else:
+            return {}
+
+    def from_relation_member(self, member):
+        stop_id = member.ref
+
+        if isinstance(member, overpy.RelationNode):
+            stop_type = "node"
+        elif isinstance(member, overpy.RelationWay):
+            stop_type = "way"
+        else:
+            raise RuntimeError("Unknown type: " + str(member))
+
+        return Stop(stop_id, stop_type)
+
+class BaseRoute(object):
```

Comment:
Yes, in order to keep this PR as small as possible, II was not changing any of the logic or finer structure. Things like these are going to be handled in #26.